static/js/echo.js

```diff
@@ -367,13 +364,11 @@ export function update_message_lists({old_id, new_id}) {
     if (all_messages_data !== undefined) {
         all_messages_data.change_message_id(old_id, new_id);
     }
-    for (const msg_list of [message_lists.home, message_list.narrowed]) {
-        if (msg_list !== undefined) {
-            msg_list.change_message_id(old_id, new_id);
+    for (const msg_list of message_lists.all_rendered_message_lists()) {
+        msg_list.change_message_id(old_id, new_id);
 
-            if (msg_list.view !== undefined) {
-                msg_list.view.change_message_id(old_id, new_id);
-            }
+        if (msg_list.view !== undefined) {
```

Comment:
These checks for the view being undefined have been useless since we downgraded `all_messages_data` to a MessageListData object. Same for some of the `table_name` checks. I'll fix this in a separate new commit.